website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -135,8 +135,8 @@ To query metric values, here are the following parameters that are available:
 
 | Parameter | Description  | Example    | Type |
 | --------- | -----------| ------------ | -------------------- |
-| `metrics`   | The metric name as defined in your dbt metric configuration   | `metrics=[revenue]` | Required    |
-| `group_by`  | Dimension names or entities to group by. We require a reference to the entity of the dimension (other than for the primary time dimension), which is pre-appended to the front of the dimension name with a double underscore. | `group_by=[user__country, metric_time]`     | Optional   |
+| `metrics`   | The metric name as defined in your dbt metric configuration   | `metrics=['revenue']` | Required    |
+| `group_by`  | Dimension names or entities to group by. We require a reference to the entity of the dimension (other than for the primary time dimension), which is pre-appended to the front of the dimension name with a double underscore. | `group_by=['user__country', 'metric_time']`     | Optional   |
 | `grain`   | A parameter specific to any time dimension and changes the grain of the data from the default for the metric. | ```group_by=[`Dimension('metric_time').``` <br/> ```grain('week\|day\|month\|quarter\|year')]``` | Optional     |
```

Comment:
@rpourzand should "Dimension" not have the ` as a prefix? So should it be just Dimension without any quotes?

```suggestion
| `grain`   | A parameter specific to any time dimension and changes the grain of the data from the default for the metric. | `group_by=[Dimension('metric_time')` <br/> `grain('week\|day\|month\|quarter\|year')]` | Optional     |
```

Reply:
Yeah, Dimension doesn't need quotes just the objects within the parens!